src/Client.ts

```diff
@@ -20,9 +19,11 @@ import { DDiscord, DOption, DSlash } from "./decorators";
 import { GuildNotFoundError } from "./errors";
 
 export class Client extends ClientJS {
+  botid?: string;
```

Comment:
First off, this should be `botId` since it's two words. Could also just be `id`.
Secondly, this should be private and have a public getter and setter.
```suggestion
  private _botId?: string;
```

Reply:
okay